how-to-be-a-developer/readme.md

```diff
@@ -16,6 +17,8 @@ At New Atoms we work in teams because we think better things are made together.
 
 As a team member your job is not to make the best things or do the most work. It's to make the whole team as productive as possible.
 
+With productive we do not mean to produce as much in as little time as possible, whatever the cost. With productive we mean that teams and its team members have the proper enviroment and stimulation to do the best and most they can for as long as possible. 
+
```

Comment:
maybe emphasize ```do not```?

" to do the best and most they can for as long as possible. "
Don't think this is informative enough.